modules/openapi-generator/src/main/resources/csharp/libraries/generichost/modelGeneric.mustache

```diff
@@ -204,6 +204,7 @@
         {{/isNew}}
         {{/isInherited}}
         {{^isInherited}}
+        {{! {{^vendorExtensions.x-composed-override}}
```

Comment:
thanks for the PR

is `{{!` (for comment) supposed to be included in this line?

Reply:
It was not meant to get committed. Removed.

Reply:
👌 

when you've time, please PM me via Slack. thank you.